### ISSUE TITLE
Create unique URL for new script editor to differentiate from the old data explorer

### ISSUE THREAD

**Issue by juliajames12** [influxdata/ui#6022]
Daniel and I would like to gather heatmap data of the new script editor using VWO, but in order to do so we need the new script editor to have its own unique URL. 

I don't have a strong preference but we could call it /fluxscripteditor

**Comment by juliajames12** [comment]
@garylfowler Let me know if you think /fluxscripteditor would be a good name.

**Comment by garylfowler** [comment]
Yes, until we had SQL, but maybe that can be its own name as well.
